aws.go

```diff
@@ -129,11 +133,20 @@ func scrape(collector *cwCollector, ch chan<- prometheus.Metric) {
 				NextToken: nextToken,
 			})		
 			if err != nil {
```

Comment:
Move it up to before the continue statement on error.
```suggestion
                        totalRequests.Inc()
			if err != nil {
```